.github/workflows/test.yml

```diff
@@ -15,10 +15,8 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  #### TEST STAGE ####
+  #### TEST JOB ####
```

Comment:
Stages are in Travis CI :)

Reply:
Irrelevant change. This is a left-over of when the CI was converted from Travis to GHA. The comment could even be removed.